pages/projects/index.html

```diff
@@ -108,6 +108,24 @@ <h2 class="project-title">SaiMan</h2>
           </div>
         </div>
       </div>
+      <div class="project-box">
+        <!-- Recommended Thumbnail Aspect Ratio 5:3 -->
+        <img class="project-thumbnail" src="./img/projects/project-HinglishEval.png">
+        <div class="project-info">
+          <h2 class="project-title">HinglishEval</h2>
+          <p> Hinglish is a blend of Hindi and English, with frequent usage of English words in sentences with standard Hindi grammar. The HinglishEval project evaluates the effectiveness of code-generation models on Hinglish prompts.  </p>
+          <div class="project-buttons">
+            <a class="project-button" href="https://sathvik040105.github.io/crypto-SaiMan/">
```

Comment:
You can link https://github.com/mrigankpawagi/HinglishEval/blob/main/draft/HinglishEval.pdf for now. We will update this with the Springer link later.

Reply:
fixed, and made the other changes too